Source/WebCore/platform/graphics/gstreamer/eme/WebKitOpenCDMDecryptorGStreamer.cpp

```diff
@@ -202,6 +230,23 @@ static bool webKitMediaOpenCDMDecryptorDecrypt(WebKitMediaCommonEncryptionDecryp
         return false;
     }
 
+#ifdef USE_OPENCDM_BCM_NEXUS_SVP
+    B_Secbuf_Info secureBufferInfo;
+    void* opaqueData;
+
+    // If there is no subsample, only allocate one region for clear+enc, otherwise, number of subsamples.
+    uint32_t rpcSubSampleTotalSize = (subSampleCount ? subSampleCount * 2 : 2) * sizeof(uint32_t);
+    uint32_t sizeOfRPCInfo = sizeof(RPCSecureBufferInformation) + rpcSubSampleTotalSize;
```

Comment:
subSampleCount could be a 32-bit number, so we could get an overflow in rpcSubSampleTotalSize. While in practice it's rare to find samples with > 10 subsamples, it's better to be safe that sorry, since that would be a very hard to track down bug if we got unusual input. Better to handle it with an appropriate type, or assert there's no overflow at least.